views/js/controller/Delivery/monitoring.js

```diff
@@ -923,6 +923,9 @@ define([
                                     remaining = 0;
```

Comment:
Check, please, that `rowTimer.consumedExtraTime` exists on the previous condition